.cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This seems like something we might not want under version control. Actually, what the heck is the `.cache` folder anyways?

Reply:
Hmmmm....I don't know. It does not seem to be meaningful, so I'll remove it.